tests/integration/100.command.dev.test.cjs

```diff
@@ -781,6 +770,53 @@ test('should respect in-source configuration from edge functions', async (t) =>
           handler: () => new Response('Hello world'),
           name: 'hello',
         })
+        .buildAsync()
+
+      await waitForLogMatching('Reloaded edge function')
+
+      const res2 = await got(`http://localhost:${port}/hello-1`, { throwHttpErrors: false })
+
+      t.is(res2.statusCode, 404)
+
+      const res3 = await got(`http://localhost:${port}/hello-2`, { throwHttpErrors: false })
+
+      t.is(res3.statusCode, 200)
+      t.is(res3.body, 'Hello world')
+    })
+  })
+})
+
+test('should respect in-source configuration from internal edge functions', async (t) => {
```

Comment:
@eduardoboucas do you have an idea what I might have done wrong in these two tests? I keep getting failed tests in the pipeline (running it locally is useless, they are incredibly flaky) and I'm having a hard time figuring out the bug I introduced.